src/ptvsd/adapter/server.py

```diff
@@ -60,6 +61,10 @@ def initialize(self, request):
         request.wait_for_response()
         self.capabilities = self.Capabilities(self, request.response)
 
+    def set_debugger_property(self, **kwargs):
```

Comment:
I'd rather not use kwargs for this - you get unfortunate intersection with Python keywords not being valid keys etc. Let's stick to dicts for everything that has to go through JSON as is.